app/models/assignment.rb

```diff
@@ -51,10 +52,19 @@ class Assignment < ActiveRecord::Base
     ocaml: "ocaml"
   }
 
+  FILE_STRUCTURES = {
```

Comment:
Add comment as to what this FILE_STRUCTURES enum is doing?

Reply:
I have added some comments.